abs.yaml

```diff
@@ -0,0 +1,3 @@
+channels:
+- psteyer-tests
+upload_without_merge: True
```

Comment:
This _is_ heading to Snowflake, right?

Reply:
Yup! I've removed the abs information and added back in snowflake. This was just for me to test to make sure everything builds correctly.